usagov/usagov-2021
# USAGOV-660-Question-Spanish-Phone-Link: Update haveaquestion.html.twig

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-660

## Description

Added language check to phone link and Spanish link when applicable.
### Testing Instructions
- Go to a page where the "Do You Have a Question" box is. Check that the phone link points to /phone. 
- Switch to Spanish page. Check that the phone link points to /es/centro-de-llamadas.

## Checklist for the Developer



## Checklist for the Peer Reviewers
